src/Symfony/Component/Form/Resources/translations/validators.th.xlf

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="th" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="28">
+                <source>This form should not contain extra fields.</source>
+                <target>ฟอร์มนี้ไม่ควรมี extra fields</target>
+            </trans-unit>
+            <trans-unit id="29">
+                <source>The uploaded file was too large. Please try to upload a smaller file.</source>
+                <target>ไฟล์ที่อัพโหลดมีขนาดใหญ่เกินไป กรุณาลองอัพโหลดใหม่อีกครั้งด้วยไฟล์ที่มีขนาดเล็กลง</target>
+            </trans-unit>
+            <trans-unit id="30">
+                <source>The CSRF token is invalid. Please try to resubmit the form.</source>
+                <target>CSRF token ไม่ถูกต้อง กรุณาลองส่งแบบฟอร์มใหม่</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>ค่านี้ไม่ใช่ค่าที่ถูกต้องของค่าสี HTML5</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>กรุณากรอกวันเดือนปีเกิดที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>ตัวเลือกที่เลิอกไม่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>คอเล็กชั่นไม่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>กรุณาเลือกค่าสีที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>กรุณาเลือกประเทศที่ถูกต้อง</target>
```

Comment:
Does this say "valid" or "correct"?

Reply:
It used the same term in Thai. 

You can double check by use Google Translate "Valid" to Thai then swap result to English again.
Valid -> ถูกต้อง, ถูกต้อง -> Correct

Reply:
Cool cool. Im just making sure.